sklearn/impute/tests/test_impute.py

```diff
@@ -971,6 +971,59 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
+@pytest.mark.parametrize("min_value, max_value",
+                         [(0, 100), (None, None), (-np.inf, np.inf),
```

Comment:
I would not support None. Basically one have to build the array by hand and put `None` inside. Putting `np.inf` or `-np.inf` will be the same assle. It would simplify greatly the code on our side.